changelogs/changelog.yml

```diff
@@ -1,5 +1,62 @@
 ancestor: null
 releases:
+  2.3:
+    changes:
+      major_changes:
+      - Add support Pardus 21.x autoinstall
+      - Add support FreeBSD 13 or later autoinstall
```

Comment:
Add "." at each end of line for all the new added lines.

Reply:
Fixed.